test/client/fixtures/sandbox/event/event-test.js

```diff
@@ -301,3 +302,31 @@ test('SVGElement.dispatchEvent should be overriden (GH-614)', function () {
 
     svg.parentNode.removeChild(svg);
 });
+
+test('should not wrap invalid event handlers (GH-1251)', function () {
+    var handlers = [void 0, 1, null, 'str', {}];
+
+    // NOTE: some handlers in some browsers throw the "Invalid argument" error
```

Comment:
Don't exclude handlers that raise exception from testing.
Let's change test scenario to:
- memory handlers that raises an error (nativeAddEventListener)
- add handler to overriden addEventListener. If it raises an error then try to find approprite error in stored errors. If it called without error then check the `listeningCtx`.

Also add `Symbol` type checking (if browser supports this type)